.circleci/config.yml

```diff
@@ -56,6 +53,19 @@ aliases:
     environment:
       DD_REQUEST_DUMPER_FILE: dump.json
 
+  - &IMAGE_DOCKER_TEST_AGENT
+    image: williamconti549/dd-apm-test-agent:latest
```

Comment:
Why not use the "official" test agent?

Reply:
Oh I saw the description.  So can we wait for the PR to be merged on the test agent side ?

Reply:
yes exactly, had to make changes so I need to host my own image

Reply:
I've switched this back to use the official image, ready for review.